src/builder.cr

```diff
@@ -34,6 +37,18 @@ module Mint
         end
       end
 
+      unless artifacts.assets.empty?
+        FileUtils.mkdir Path[DIST_DIR, ASSET_DIR].to_s
+
+        terminal.measure "#{COG} Writing assets..." do
+          artifacts.assets.each do |asset|
+            File.write Path[DIST_DIR, ASSET_DIR, asset.filename.to_s], asset.real_path
+          end
+
+          File.write Path[DIST_DIR, "external-stylesheets.css"], SourceFiles.external_stylesheets
```

Comment:
Shouldn't this be left outside of the `unless` branch?

Reply:
Yes, I was planning to fix this later :joy: